Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.38660&attempt_id=684265ed-e939-4ef1-bd2a-9e9da31b08b4)

Attempted: btrbk

<details><summary>Partial log (click to expand)</summary><p>

```
build flags: SHELL=/nix/store/q1g0rl8zfmz7r371fp5p42p4acmv297d-bash-4.4-p19/bin/bash
generating manpages...
make[1]: Entering directory '/build/btrbk-0.26.1/doc'
a2x -L -d manpage -f manpage btrbk.1.asciidoc
a2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/nix/store/7f2hh08wyl2zjxz9p02d2s63b8a31nfp-asciidoc-8.6.9/etc/asciidoc/docbook-xsl/manpage.xsl" "/build/btrbk-0.26.1/doc/btrbk.1.xml" returned non-zero exit status 5
make[1]: *** [Makefile:46: btrbk.1] Error 1
make[1]: Leaving directory '/build/btrbk-0.26.1/doc'
make: *** [Makefile:76: man] Error 2
builder for '/nix/store/kfrkz468722bad67p9l0abf5cd9571ap-btrbk-0.26.1.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/kfrkz468722bad67p9l0abf5cd9571ap-btrbk-0.26.1.drv' failed
```
</p></details>

--- a/pkgs/tools/backup/btrbk/default.nix
+++ b/pkgs/tools/backup/btrbk/default.nix
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, coreutils, bash, btrfs-progs, openssh, perl, perlPackages
-, utillinux, asciidoc-full, makeWrapper }:
+, utillinux, asciidoc, makeWrapper }:
 
 stdenv.mkDerivation rec {
   name = "btrbk-${version}";
@@ -10,7 +10,9 @@ stdenv.mkDerivation rec {
     sha256 = "04ahfm52vcf1w0c2km0wdgj2jpffp45bpawczmygcg8fdcm021lp";
   };
 
-  buildInputs = with perlPackages; [ asciidoc-full makeWrapper perl DateCalc ];
+  nativeBuildInputs = [ asciidoc makeWrapper ];
+
+  buildInputs = with perlPackages; [ perl DateCalc ];
 
   preInstall = ''
     for f in $(find . -name Makefile); do

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -886,7 +886,9 @@ with pkgs;
 
   btrfs-dedupe = callPackage ../tools/filesystems/btrfs-dedupe/default.nix {};
 
-  btrbk = callPackage ../tools/backup/btrbk { };
+  btrbk = callPackage ../tools/backup/btrbk {
+    asciidoc = asciidoc-full;
+  };
 
   buildtorrent = callPackage ../tools/misc/buildtorrent { };
 

